### PR TITLE
Update help.html

### PR DESCRIPTION
**Pull request by ghost**
Some fixes for French language.

### CODE REVIEW

File: Padiddle/Padiddle/fr.lproj/help.html [hunk 1]
```diff
@@ -106,18 +106,18 @@
     </head>
     <body>
         <h1>Instructions</h1>
-        <p>Touchez le bouton <img id="recordButton" src="asset://recordButton" alt="Image du bouton Enregistrement de l&rsquo;application" />, puis tournez votre ^deviceName^ comme suit:</p>
+        <p>Touchez le bouton <img id="recordButton" src="asset://recordButton" alt="Image du bouton Enregistrement de l&rsquo;application" />, puis tournez votre ^deviceName^ comme suit :</p>
```
Comment (ZevEisenberg) [inline Padiddle/Padiddle/fr.lproj/help.html:109]: Is the space at the end of this line intentional?

File: Padiddle/Padiddle/fr.lproj/help.html [hunk 2]
```diff
@@ -106,18 +106,18 @@
...
         <p><img id="deviceImage" src="asset://^deviceImage^" alt="Appareil iOS tournant sur le doigt d&rsquo;une personne" /></p>
-        <p>Touchez le bouton <img id="colorButton" src="asset://colorButton" alt="Image du bouton Couleur de l&rsquo;application" /> pour choisir un thème de couleur dans le biliotheque.</p>
+        <p>Touchez le bouton <img id="colorButton" src="asset://colorButton" alt="Image du bouton Couleur de l&rsquo;application" /> pour choisir un th&egrave;me de couleur dans la bilioth&egrave;que.</p>
         <h1>Support</h1>
-        <p>Questions? Commentaires? Suggestions? Je voudrais vous entendre! Contactez-moi sur <a href="https://twitter.com/ZevEisenberg">Twitter</a> et consultez les FAQ sur le <a href="http://ZevEisenberg.com/#padiddle-faq">site</a>.</p>
+        <p>Des questions ? Commentaires ? Suggestions ? Je voudrais vous entendre ! Contactez-moi sur <a href="https://twitter.com/ZevEisenberg">Twitter</a> et consultez la FAQ sur le <a href="http://ZevEisenberg.com/#padiddle-faq">site</a>.</p>
```
Comment (ZevEisenberg) [inline Padiddle/Padiddle/fr.lproj/help.html:113]: Are the spaces before the punctuation marks intentional?

**Review by ZevEisenberg — COMMENTED** [review]
@Relisiuol thank you for the updates! A couple of minor questions.

**Comment by ZevEisenberg** [comment]
@Relisiuol thanks for your contribution! I don't believe the HTML entities are needed, since the encoding is UTF-8 and I'm rendering the document in the app, so there's no worry that some other browser will get it wrong. I incorporated the rest of your changes in this commit: ad5830091c40d2c41d27b4bb5602198655f45f00

**Comment by Relisiuol** [comment]
You’re welcome and yes as you can read in Wikipedia (page Punctuation):
In French of France, the signs : ; ? and ! are always preceded by a thin unbreakable space.

**Comment by ghost** [comment]
You’re welcome and yes as you can read in Wikipedia (page Punctuation):
In French of France, the signs : ; ? and ! are always preceded by a thin unbreakable space.

**Comment by ZevEisenberg** [comment]
Got it. I added `&nbsp;` before those punctuation marks here: 4236664

(I tried to get thinner non-break spaces to work as described [here](http://jkorpela.fi/html/french.html), but I couldn't get it.)

**Pull request closed without merging**